### ISSUE TITLE
WP Side Comments Compatibility Recommendation

### ISSUE THREAD

**Issue by michaelbeil** [AesopInteractive/lasso#17]
What if linked to the WP Side Comments add-on upon setup wizard? https://github.com/AesopInteractive/lasso-wp-side-comments

![screen shot on 2015-03-26 at 11_26_45](https://cloud.githubusercontent.com/assets/4752949/6851176/63e3d5fa-d3ab-11e4-9e42-a0f06f96bf56.png)

**Comment by bearded-avenger** [comment]
good idea. we just need to test it more. When i was testing it locally, and I refreshed the page after saving, the comments werent showing. So although it works, something is weird with it.
